src/annotator/hypothesis-injector.js

```diff
@@ -43,31 +43,33 @@ export class HypothesisInjector {
   /**
    * Inject Hypothesis client into a newly-discovered iframe.
    *
+   * IMPORTANT: This method requires that the iframe is "accessible"
+   * (frame.contentDocument|contentWindow is reachable) and "ready" (DOM content
+   * is interactive) before the method is called.
```

Comment:
This is better - I think I would suggest to replace "reachable" by "not null", to be more specific and "is interactive" by "has been loaded". I think it is ambiguous whether the document is considered "interactive" at the point when `DOMContentLoaded` has fired but `load` has not (eg. on a website with many images, the page might not be useful from the user's point of view at this point, but Hypothesis can still function).